project/Dependencies.scala

```diff
@@ -6,13 +6,13 @@ object Dependencies {
     val scala211 = "2.11.12"
     val scala212 = "2.12.12"
     val scala213 = "2.13.3"
-    val scala30 = "3.0.0-M3"
+    val scala30 = "3.0.0-RC1"
 
     val shapeless = "2.3.3"
     val typesafeConfig = "1.4.1"
 
-    val scalaTest = "3.2.3"
-    val scalaTestPlusScalaCheck = "3.2.2.0"
+    val scalaTest = "3.2.5"
```

Comment:
Having dependencies structured in this non-standard way prevents scala-sweward from detecting updates and bumping versions. I don't think we gain anything in not inlining them, I'll investigate changing it.

Reply:
What do you mean by non-standard way? Scala Steward seemed to be able to propose version updates in the past for these dependencies, #816 and #865 being examples of it.

Reply:
Oh, never mind then. Non-standard way as in not in the format `"org" % "artifact" % "version"`, but it looks like scala-steward is smarter than that :)